LICENSE.txt

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2015, Jesse Berger (user alias: valadian)
```

Comment:
I don't know much about handling licenses. I thought about changing it, but decided to just leave it like that to be safe.